plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/block.tsx

```diff
@@ -172,6 +172,10 @@ const Block = ( {
 		'shippingCostRequiresAddress',
 		false
 	);
+	const localPickupTextFromSettings = getSetting< string >(
+		'localPickupText',
+		localPickupText || defaultLocalPickupText
+	);
```

Comment:
Thanks for the suggestion, it makes sense but I feel this is out of scope for this PR and changing it would increase testing. We could address that in a refactor in future.